endpoints-control/src/main/java/com/google/api/control/model/OperationInfo.java

```diff
@@ -20,12 +20,20 @@
 import com.google.api.servicecontrol.v1.Operation;
 import com.google.api.servicecontrol.v1.Operation.Importance;
 import com.google.common.base.Strings;
+import com.google.common.collect.Maps;
 import com.google.protobuf.Timestamp;
 
+import java.util.Map;
+
 /**
  * Holds basic information about an operation to be obtained from the HTTP layer.
  */
 public class OperationInfo {
+  public static final String SCC_CALLER_IP = "servicecontrol.googleapis.com/caller_ip";
```

Comment:
Might make sense to assign the common prefix "servicecontrol.googleapis.com/" to a constant, but it's not a big deal